draft-ietf-dprive-xfr-over-tls.xml

```diff
@@ -362,13 +360,19 @@ applicable to XFR-over-TLS as well.
 <t>It is RECOMMENDED that clients and servers that support XoT also implement
 EDNS0 Keepalive [RFC7828].
 </t>
+<t>It is useful to note that in these mechanisms is it the secondary that initiates the TLS connection to the primary for a XFR request, so that in terms of connectivity the secondary is the TLS client and the primary the TLS server.
```

Comment:
"It is useful to note that in these mechanisms is it" -> it is

Reply:
Fixed - thanks